help/en/html/doc/Technical/gitadmin.shtml

```diff
@@ -46,20 +46,41 @@
       </ul>
       
       <h2>Merging a PR</h2>
-      
-      Except in exceptional circumstances (which should probably be discussed in the PR itself),
-      PRs should only be merged when:
+
+      JMRI's Github is configured with the following automations and requirements for merging a Pull Request:
       <ul>
-        <li>They have passed all the CI tests</li>
-        <li>They have no outstanding reviews</li>
-        <li>If in a new area, large or from a new contributor, 
-            they have been in-queue for at least a day or so to allow people to take a look at them</li>
-        <li>Don't appear to have any un-resolved controversies in the comments</li>
+        <li>All CI tests have passed for the Pull Request</li>
+        <li>There has been at least one review from a JMRI developer that accepts the Pull Request</li>
+        <li>There are no reviews requesting changes</li>
```

Comment:
I suggest an additional point:
`<li>If there are a review requesting changes, but the author and the reviewer cannot resolve their dispute, then ????????</li>`

Note: I write question marks since I'm not sure how this should be handled.

I agree with @rhwood that some PRs should never be merged, for different reasons, but I'm also concerned if a single reviewer can veto a PR that the rest of us wants to be merged.